providers/neo4j/tests/unit/neo4j/operators/test_neo4j.py

```diff
@@ -19,7 +19,11 @@
 from unittest import mock
 
 from airflow.providers.neo4j.operators.neo4j import Neo4jOperator
-from airflow.utils import timezone
+
```

Comment:
Since we are changing the import here - it might be a good idea to already switch to lazy_compat and add dependency to common.compat in the provider (see https://github.com/apache/airflow/pull/56793)